.github/workflows/build-common.yml

```diff
@@ -238,9 +238,7 @@ jobs:
           - 11
           - 17
           - 21
-          - 23
```

Comment:
we shouldn't have been testing on 23 anymore anyways since not LTS and not latest